src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/ubuntu-dotnet-local-install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Argument = -d destdir
+usage()
+{
+cat << EOF
+usage: $0 options
+
+This script installs dotnet cli on the linux machine
+
+OPTIONS:
+   -h      Show this message
+   -d      directory where dotnet cli will be installed
+
+EOF
+}
+
+DESTDIR=
+while getopts “hd:” OPTION
+do
+     case $OPTION in
+         h)
+             usage
+             exit 1
+             ;;
+         d)
+             DESTDIR=$OPTARG
+             ;;
+         ?)
+             usage
+             exit
+             ;;
+     esac
+done
+
+echo $DESTDIR
+if [[ -z $DESTDIR ]]
+then
+     usage
+     exit 1
+fi
+
+echo "Initiating local dotnet install"
+wget https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh
```

Comment:
If my understanding of this line is correct, it's fetching a hot copy of this script, which might cause runs to start failing unexpectedly, even running the same thing a second time might produce different results (if there is a new latest change in this branch).  Picking a particular version will lead to more stable and reliable tests.